CHANGES

```diff
@@ -1,6 +1,7 @@
 v2.1.0, 2020-02-25
-- Add the possibility to provide cookies as dict to authenticate
+- Add the possibility to provide cookies as dict to authenticate (#60)
 - Add 'Referer' header for CSRF check
```

Comment:
Isn't it the same #60 ? It seems to be this line: https://github.com/CZ-NIC/python-rt/commit/0d2a32a4af5db6a97421078b3fbb0ee5e122c2b3

```python
self.session.headers.update({'referer': url})
```

Will you add it please then?

Reply:
Ahh, I only looked at the commit messages :) Added that.